dom-event-delegation/main.js

```diff
@@ -0,0 +1,11 @@
+var $taskList = document.querySelector('.task-list');
+
+$taskList.addEventListener('click', function (event) {
+  console.log('event.target:', event.target);
+  console.log('event.target.tagName:', event.target.tagName);
+  if (event.target && event.target.nodeName === 'BUTTON') {
+    var $closestElement = event.target.closest('.task-list-item');
+    console.log($closestElement);
+    $closestElement.remove($closestElement);
```

Comment:
the remove method does not take an argument, it is just called on the element you want to remove.

Reply:
gotcha! i removed it! thanks Brett